src/psyclone/tests/test_files/dynamo0p3/infrastructure/fs_continuity_mod.f90

```diff
@@ -0,0 +1,50 @@
+!-----------------------------------------------------------------------------
+! Copyright (c) 2017,  Met Office, on behalf of HMSO and Queen's Printer
+! For further details please refer to the file LICENCE.original which you
+! should have received as part of this distribution.
+!-----------------------------------------------------------------------------
+! BSD 3-Clause License
+!
+! Modifications copyright (c) 2017, Science and Technology Facilities Council
```

Comment:
Sorry for crashing into your review, @arporter, but there may be an issue with adding this file. It makes PSyclone version of LFRic infrastructure consistent with LFRic, as this is where the "regular" function spaces are defined (see [LFRic fs_continuity_mod.F90](https://code.metoffice.gov.uk/trac/lfric/browser/LFRic/trunk/infrastructure/source/function_space/fs_continuity_mod.F90?rev=23484)). However, I noticed the PSyclone version of `argument_mod.F90` is not modified in this PR which means we have duplicate FS IDs. If you look at the LFRic trunk, [argument_mod.F90](https://code.metoffice.gov.uk/trac/lfric/browser/LFRic/trunk/infrastructure/source/field/argument_mod.F90?rev=23484) sets IDs only for `ANY_*` spaces. This also impacts my PR #760 as I should really be adding `W2*trace` IDs to `fs_continuity_mod` as it is done in LFRic infrastructure.  

LFRic kernels `use fs_continuity_mod, only: ...` for the "real" function spaces and `use argument_mod, only: ...` for other metadata, including `ANY_*` spaces. This should be really done in PSyclone test and example kernels for LFRic API if we want to be consistent with LFRic infrastructure definitions and use.  I understand this is a lot of test and example kernels to modify in this PR so there could be a follow-up issue to do that whilst for now duplicate FS IDs definitions can be removed from the `argument_mod.F90` and `use fs_continuity_mod` added so that test compilation works. It would be good, though, to have this consistent with LFRic for the upcoming PSyclone release if we are already going down this route.

Reply:
I've done this and created a new issue (#764)